ide/editor.codetemplates/apichanges.xml

```diff
@@ -84,6 +84,18 @@ is the proper place.
     <!-- ACTUAL CHANGES BEGIN HERE: -->
 
     <changes>
+        <change id="CodeTemplateFilter.Factory.createForDoc">
+             <api name="codetemplates"/>
+             <summary>Default method <code>createFilter(Document, int, int)</code> added to <code>CodeTemplateFilter.Factory</code>.</summary>
+             <version major="1" minor="57"/>
+             <date day="9" month="9" year="2021"/>
+             <author login="dbalek"/>
+             <compatibility addition="yes" binary="compatible" deletion="no" deprecation="no" modification="no" semantic="compatible" source="compatible"/>
+             <description>
+                 Added method allows for creating code template filters for the given document and range.
```

Comment:
"allows for"? Isn't just "allows creating" better?

Reply:
Fixed.